numpy/ma/tests/test_extras.py

```diff
@@ -1565,6 +1565,21 @@ def test_atleast_2d(self):
         assert_equal(a.mask.shape, a.data.shape)
         assert_equal(b.mask.shape, b.data.shape)
 
+    def test_atleast_nd(self):
+        # Test atleast_nd
+        a = masked_array([0, 1, 2], mask=[0, 1, 0])
+        b = atleast_nd(a, 4)
+        c = atleast_nd(a, 4, pos='end')
```

Comment:
Should be `right`?

Reply:
Yes. I'm fixing the tests now. Pretty much didn't look at the tests besides borking the imports.